### ISSUE TITLE
K2A shouldn't ask to save the search data when using Accessibility plugin with browsers.

### ISSUE THREAD

**Issue by kizernis** [PhilippC/keepass2android#1807]
I use the Accessibility plugin with browsers like Bromite or Opera (that don't support native autofill) and I love it, even though I always have to push the search button and enter a few letters to find the right KeePass entry. The problem is that K2A always asks me if I want to save the search data for the current app. If I accidentally tap Yes, K2A starts to fill the same username and password for EVERY website that I open in that browser, until I manually delete the search data from that entry. Perhaps it's a good thing to add the "Never for this app" button to the prompt?

**Comment by NumbGnat** [comment]
I agree. I don't like the fact that KP2A asks if you want to save the association when the app is Chrome, Firefox, Brave, etc. It would never make sense to me, because that association would apply to any web site.

Adding a "never ask for this app" option, would solve that problem.
